src/openzaak/templates/admin/nlx_config.html

```diff
@@ -0,0 +1,61 @@
+{% extends "admin/base_site.html" %}
+{% load i18n static %}
+
+{% block extrastyle %}{{ block.super }}
+    <link href="{% static 'libs/prism/prism.css'%}" rel="stylesheet" />
+
+{% endblock %}
+
+
+{% block title %} {% trans "Importeer catalogus" %} {{ block.super }} {% endblock %}
+
+{% block breadcrumbs %}
+<div class="breadcrumbs">
+<a href="{% url 'admin:index' %}">{% trans 'Home' %}</a>
+&rsaquo; <a href="#">{% trans 'Configuratie' %}</a>
+&rsaquo; {% trans 'NLX configuratie' %}
+</div>
+{% endblock %}
+
+{% block content %}
+    <h1>{% trans 'NLX configuratie' %}</h1>
+    <div id="content-main">
+        <form action="" method="POST" enctype="multipart/form-data">
+            <div>
+                <fieldset class="module aligned ">
+
+                    <p>
+                        Gebruik onderstaand voorbeeld voor de configuratie van de NLX-inway.
+                    </p>
+                    <p>
+                        U kunt uw <code>service-config.toml</code> bestand aanpassen met de configuratie onder <code>[services]</code>.
+                    </p>
+
+                    <pre class="language-toml"><code>[services]
```

Comment:
I'm curious if using https://pypi.org/project/django-pygments/ wouldn't be a better option than adding a frontend library? Pygments is used by Sphinx as well, so it'll resemble the documentation more.

Reply:
The last update of `django-pygments` was 6 years ago and their site doesn't work. So actually I don't want to use them

Reply:
good call, I saw the latest release on PyPI from November 2019, but the latest work in the repo is effectively from 6 years ago :+1: